File: Ruby/easy/easy2/tip_calculator.rb
# Tip Calculator
puts "What is the bill?"
bill = gets.chomp.to_f

puts "What is the tip percentage?"
percent = gets.chomp.to_f

tip = (percent * bill) / 100

# Result
puts "The tip is $#{tip}"
puts "The total is $#{bill + tip}"

File: javascript/js-101-small-problems/easy1/tip-calculator.js
let rlSync = require("readline-sync");

let bill = parseFloat(rlSync.question('What is the bill? '));

let percentage = parseFloat(rlSync.question('What is the tip percentage? '));

let tip = bill * (percentage / 100);

let total = bill + tip;

console.log(`The tip is $${tip.toFixed(2)} \nThe total is $${total.toFixed(2)}`);
